Scripts/SL-Helpers-GrooveStats.lua

```diff
@@ -442,7 +442,8 @@ ValidForGrooveStats = function(player)
 	)
 
 	-- only FailTypes "Immediate" and "ImmediateContinue" are valid for GrooveStats
```

Comment:
I would add a comment stating why po:FailSetting doesn't work (something akin to, this setting can be overwritten by preferences so to guarantee the setting we use GetPlayerFailType)